test/e2e/framework/expect.go

```diff
@@ -35,9 +35,15 @@ func ExpectError(err error, explain ...interface{}) {
 	gomega.ExpectWithOffset(1, err).To(gomega.HaveOccurred(), explain...)
 }
 
-// ExpectNoError checks if "err" is set, and if so, fails assertion while logging the error.
-func ExpectNoError(err error, explain ...interface{}) {
-	ExpectNoErrorWithOffset(1, err, explain...)
+// ExpectNoError checks if "err" is set, and if so, fails the assertion while logging the error.
+//
+// An additional printf-style explanation must be provided because often errors
+// cannot be understood without it. In cases where the error is known to
+// be descriptive enough in all cases, an empty explainFormatStr can be
+// passed.
+func ExpectNoError(err error, explainFormatStr string, explainArgs ...interface{}) {
+	explainArgs = append([]interface{}{explainFormatStr}, explainArgs...)
+	ExpectNoErrorWithOffset(1, err, explainArgs...)
```

Comment:
Please also update ExpectNoErrorWithOffset, ExpectErrorWithOffset, and ExpectError accordingly.

Beware that the implementation needs an if check for the case where `explainFormatStr` is empty: in that case we don't want to pass any additional arguments to Gomega.